### ISSUE TITLE
Rest v2 "Get a Table" endpoint does not correctly report defaultTimeToLive

### ISSUE THREAD

**Issue by hemidactylus** [stargate/stargate#1836]
**Overview**

When retrieving the table properties with the Rest v2 endpoint GET `/v2/schemas/keyspaces/{keyspace-id}/tables/{table-id}`, the returned value for `tableOptions.defaultTimeToLive` (which should correspond to the underlying `default_time_to_live` in CQL language) is always returned as `0`, regardless of the actual table settings as seen in CQL.

This is observed using Astra DB and the currently-running (as of 2022-05-20) version of Stargate on top of it.

**Motivation**

Users would expect a faithful mapping between table features as seen with CQL `describe table` queries and using the Rest v2 API.

**Reproduce**

1. Create a table in Astra DB with a nonzero `default_time_to_live`;
2. check with `describe table` and notice the nonzero setting being reported (also can check by watching an inserted row evaporate away!);
3. query the same table with the Rest v2 API endpoint above: there'll be `"defaultTimeToLive": 0`.

(See attachments based on Astra DB)

_Incidentally, the opposite works fine, i.e. a table created with the PUT [...] endpoint will have the intended default time to live properly set._
![def_ttl_1](https://user-images.githubusercontent.com/14221764/169534165-4a0e0ebc-5ca6-4252-abc4-9112b0a548ba.png)
![def_ttl_2](https://user-images.githubusercontent.com/14221764/169534169-c9fad924-88f1-4ffe-b4a6-6a36c668547d.png)

See images.

**Suggested fix**

Alas, I am not familiar enough with the codebase to venture into this.

**Comment by ivansenic** [comment]
@hemidactylus Thanks for reporting :+1: 

@tatu-at-datastax Wanna have a look?

**Comment by tatu-at-datastax** [comment]
@ivansenic thanks, will have a quick look. Pretty safe to assume there's no testing for accessing that option.

**Comment by tatu-at-datastax** [comment]
Ok so the code in question does indeed not fetch anything:

```
class TablesResource {
  // ...
  private TableResponse getTable(AbstractTable tableMetadata) {
    // ...
    final int ttl =
        0; // TODO: [doug] 2020-09-1, Tue, 0:08 get this from schema (select default_time_to_live
    // from tables;)
    final TableOptions tableOptions = new TableOptions(ttl, clusteringExpression);
```

**Comment by tatu-at-datastax** [comment]
Looks like SGv2 has same challenges, Table resource has:

```
    // !!! TODO: figure out where to find TTL?
    List<Sgv2Table.ClusteringExpression> clustering =
        clustering2clustering(grpcTable.getClusteringOrdersMap());
    final Sgv2Table.TableOptions tableOptions = new Sgv2Table.TableOptions(null, clustering);
```

So as things are, support for accessing `ttl` is unfortunately unsupported.

**Comment by tatu-at-datastax** [comment]
Note: fix would require first doing #1881.
